lib/src/dartdevc/errors.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:barback/barback.dart';
+
+/// An [Exception] that is thrown when the analyzer fails to create a summary.
+class AnalyzerSummaryException implements Exception {
+  /// The module that couldn't be compiled.
+  final AssetId assetId;
```

Comment:
We know it's an AssetId from the type. How about a more meaningful name. Maybe "failedModule" or just "module"?